ds701_book/jupyter_notebooks/11-Dimensionality-Reduction-SVD-II.ipynb

```diff
@@ -5,259 +5,259 @@
       "metadata": {},
```

Comment:
<p>"SVD of the data matrix" or SVD of product of the transpose of the data matrix and data matrix?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tools4ds/DS701-Course-Notes/pull/145/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>You would take the SVD of X, the eigenvalue computation with X^TX is not always numerically stable (you're squaring the condition number of the data matrix X). The vectors V are the principal components and then you can eigenvalues from \Sigma^2.</p><p> </p><p>I will add a remark to clarify this.</p>